adafruit_esp32spi/adafruit_esp32spi_wsgiserver.py

```diff
@@ -98,7 +98,7 @@ def update_poll(self):
         """
         self.client_available()
         if (self._client_sock and self._client_sock.available()):
-            self.print_remote_ip()
+            # self.print_remote_ip()
```

Comment:
sorry -- commenting out code  is not a good idea in my opinion. either remove the function or leave it in.